build-image/first-boot.bash

```diff
@@ -134,23 +134,17 @@ sleep 10  # Related to: https://github.com/openhab/openhabian/issues/441#issueco
 echo "OK"
 
 echo -n "$(timestamp) [openHABian] Updating repositories and upgrading installed packages... "
-apt-get update &>/dev/null
-/usr/bin/apt-get --yes --fix-broken install &>/dev/null
-apt-get --yes upgrade &>/dev/null
-if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
+apt-get --yes --fix-broken install &>/dev/null
+if apt-get --yes upgrade &>/dev/null; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
 
 if hash python 2>/dev/null; then bash /boot/webif.bash reinsure_running; fi
 
 echo -n "$(timestamp) [openHABian] Installing git package... "
-apt-get update &>/dev/null
-/usr/bin/apt-get -y install git &>/dev/null
-if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
+if apt-get -y install git &>/dev/null; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
 
 echo -n "$(timestamp) [openHABian] Cloning myself... "
-[ -d /opt/openhabian/ ] && rm -rf /opt/openhabian/ # check if we have remnants of a previous installation attempt.
-git clone -b master https://github.com/openhab/openhabian.git /opt/openhabian &>/dev/null
-
-if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
+if [ -d /opt/openhabian/ ]; then cd /opt && rm -rf /opt/openhabian/; fi
```

Comment:
What do `cd /opt` do here?!

Reply:
I don't recall. Right now it's obviously unneeded, then again there are cases when you change the arguments to/right hand side of the "rm" command later on to compute them (say as a result of shell expansion) this computation can become "" or "*", and in such a case it's safer to be in a known directory where this command shouldn't do harm.

Reply:
removed that as it's confusing